test/openwhisk-adapter.test.js

```diff
@@ -40,7 +48,7 @@ describe('OpenWhisk Adapter Test', () => {
           });
           return new Response(ret);
         },
-        '@noCallThru': true,
+
```

Comment:
Didn't know that 😄 Was it removed everywhere because it's unnecessary?

Reply:
it's defined above already in the import:
```
const proxyquire = require('proxyquire').noCallThru();
```